src/Security/MemberAuthenticator/LoginForm.php

```diff
@@ -82,7 +87,9 @@ public function __construct(
             $backURL = Session::get('BackURL');
         }
 
-        if ($checkCurrentUser && Member::currentUser() && Member::logged_in_session_exists()) {
+        if ($checkCurrentUser && Security::getCurrentUser() && Member::logged_in_session_exists()) {
```

Comment:
Correction, that doesn't exist in the IdentityStore. The IdentityStore can grab members from an ADFS for example, map it to our members, without writing to the database. Other than that, true.